tests/run_with_server

```diff
@@ -17,7 +17,7 @@ trap 'kill -s SIGINT $SIM_PID; wait $SIM_PID' EXIT
 
 # Wait for the server to start
 sleep 0.2
-while ! echo '*IDN?' | nc localhost 8888; do
+while ! echo '*IDN?' | nc -N localhost 8888; do
```

Comment:
I am unable to find any reference to a `-N` option to `nc`; for me it generates the message "nc: invalid option -- 'N'"

Reply:
In GNU netcat, if you don't pass that, netcat will keep the connection open when piping output from another command even when EOF is received, the flag `-N` is making netcat behaves as expected and close the connection on EOF.
I'm afraid the simplest approach to fix this is not using GNU netcat, I will try to install a different netcat implementation in the docker image and drop the `-N` commit.